lib/features/navigation_tab_view/navigation_tab_view.dart

```diff
@@ -36,8 +37,8 @@ class NavigationTabView extends ConsumerWidget {
           ),
         ),
         child2: SmallTileCard(
-          onTap: () {
-            // TODO(simon-the-shark): implement review request
+          onTap: () async {
+            await InAppReview.instance.openStoreListing();
```

Comment:
Overall looks, I didn't run it but I assume it works (you can provide some picture if that's not a problem)

I've got one thing, 
WDYT about creating such a method in `InAppRatingService` class and call it from `navigation_tab_view`? Looks like overkill but imagine if we (for some reasons) would have to change library for in app review? @simon-the-shark  @tomasz-trela wdyt?

Reply:
some images

before click:
![image](https://github.com/user-attachments/assets/ca026036-c56a-444c-b9ec-a43fa6b5e516)
after click:
![image](https://github.com/user-attachments/assets/1426e81e-6119-4b12-bd4b-abd6bb0c19c1)


didnt check that on ios tho :(

if it comes to making specified class I will wait for opinions of others and I can handle implementation if others agree on doing that!

Reply:
> Overall looks, I didn't run it but I assume it works (you can provide some picture if that's not a problem)
> 
> I've got one thing, 
> WDYT about creating such a method in `InAppRatingService` class and call it from `navigation_tab_view`? Looks like overkill but imagine if we (for some reasons) would have to change library for in app review? @simon-the-shark  @tomasz-trela wdyt?  


I think it is a good practice.

Reply:
And I think you're guys overcomplicating oneliner solution 😭😭😭

Reply:
Since've got that service already defined, it's almost no cost defining method in there. From `navigation_tab_view` perspective nothing changes and code looks cleaner, we can controll anything from one place